src/components/ContributionForm/ContributionForm.tsx

```diff
@@ -124,6 +147,14 @@ const ContributionForm = ({ submit }: { submit: (data: any) => void }) => {
           onChange={(e) => setOtherWish(e.target.value)}
         />
       )}
+      {additionWishEmpty && (
+        <p>
+          <Alert
+            type="error"
+            title={'Merci de selectionner le type d’ajout.'}
+          />
+        </p>
+      )}
```

Comment:
should be automatically managed by the required on the checkbox group ?

Reply:
Yes, I think so too, but it seems it's not and since there's no doc on the React-DSFR, I'm not sure I'm doing it right.